lib/ContainerManager.js

```diff
@@ -23,6 +25,17 @@ Promise.longStackTraces();
 
 const SETUP_CONTEXT = "env"
 
+var log_container_events = function (container){
+  var events = ["running tasks", "tasks complete", "creating", "stopping", "restarting",
```

Comment:
Arrays that don't fit on one line should just have one element per line rather tha line breaking in the middle of the array definition.